views/mail_body.tpl

```diff
@@ -5,6 +5,8 @@ You are receiving this message because you are the maintainer of aport:
 
 {{{origin}}}
 
+abump {{{origin}}}-{{{new_version}}}
```

Comment:
new_version is only available when you assign it in:

https://github.com/clandmeter/aports-turbo/blob/master/model.lua#L241